docs/architecture/cloud-native/resilient-communications.md

```diff
@@ -1,54 +1,54 @@
+- *Service discovery.* With each microservice running across a cluster of machines with its own IP address and port, how do services discover and communicate with each other?
 - *Resiliency.* How do you manage short-lived failures and keep the system stable?
-- *Load balancing.* How does inbound traffic get distributed across multiple instances of a service?
+- *Load balancing.* How does inbound traffic get distributed across multiple instances of a microservice?
 - *Security.* How are security concerns such as transport-level encryption and certificate management enforced?
-- *Distributed Monitoring.* - How do you correlate and capture traceability and monitoring for a single request across multiple consuming services?
+- *Distributed Monitoring.* - How do you correlate and capture traceability and monitoring for a single request across multiple consuming microservices?
 
-While these concerns can be addressed with various libraries and frameworks, implementing them inside your codebase can be expensive, complex, and time-consuming. Moreover, you end up with a solution where infrastructure concerns are coupled to business logic.
+You can address these concerns with different libraries and frameworks, but their implementation can be expensive, complex, and time-consuming. Moreover, you end up with a solution where infrastructure concerns are coupled to business logic.
 
 ## Service mesh
 
-A better approach is to consider a new and rapidly evolving technology entitled *Service Mesh*. A [service mesh](https://www.nginx.com/blog/what-is-a-service-mesh/) is a configurable infrastructure layer with built-in capabilities to handle service communication and many of the challenges mentioned above. It decouples these concerns from your business code and moves them into a service proxy, an instance of which accompanies each of your services. Often referred to as the [Sidecar pattern](https://docs.microsoft.com/azure/architecture/patterns/sidecar), the service mesh proxy is deployed into a separate process to provide isolation and encapsulation from your business code. However, the proxy is closely linked to the service being created along with it and sharing its lifecycle. Figure 6-9 shows this scenario.
+A better approach is an evolving technology entitled *Service Mesh*. A [service mesh](https://www.nginx.com/blog/what-is-a-service-mesh/) is a configurable infrastructure layer with built-in capabilities to handle service communication and the other challenges mentioned above. It decouples these concerns by moving them into a service mesh proxy. Known as the [Sidecar pattern](https://docs.microsoft.com/azure/architecture/patterns/sidecar), the proxy is deployed into a separate process to provide isolation from your service business code. However, the proxy is closely linked to the service. It is created along with it and shares its lifecycle. Figure 6-7 shows this scenario.
```

Comment:
What's service mesh proxy? Shouldn't it be called a sidecar proxy?

Reply:
Addressed